_rules/SC1-1-1-image-has-name.md

```diff
@@ -47,84 +47,110 @@ There is a known combination of a popular browser and assistive technology that
 
 #### Passed example 1
 
-Image has accessible name
+HTML `img` element has accessible name
 
 ```html
 <img alt="W3C logo" />
 ```
 
 #### Passed example 2
 
-Non-image element with image role and accessible name
+Element with role of `img` and accessible name
 
 ```html
 <div role="img" aria-label="W3C logo"></div>
 ```
 
 #### Passed example 3
 
-Accessible name but not always supported.
+Accessible name but not always accessibility supported
 
 ```html
 <img title="W3C logo" />
 ```
 
-### Failed
+#### Passed example 4
 
-#### Failed example 1
+HTML `img` element marked as [decorative](#decorative) through empty `alt` attribute
+
+```html
+<img alt="" />
+```
 
-No accessible name
+#### Passed example 5
+
+HTML `img` element marked as [decorative](#decorative) through `role="presentation"`
 
 ```html
-<img />
+<img role="presentation" />
 ```
 
-#### Failed example 2
+#### Passed example 6
+
+HTML `img` element marked as [decorative](#decorative) through `role="none"`
 
-Non-image element with image role but no accessible name.
 ```html
-<div role="img"></div>
+<img role="none" />
 ```
 
-#### Failed example 3
+#### Passed example 7
+
+HTML `img` element is not [included in the accessibility tree](#included-in-the-accessibility-tree) but has accessible name
 
-Image element inside a div positioned off screen with no accessible name.
 ```html
-<div style="margin-left:-9999px;"><img /></div>
+<img alt="W3C logo" aria-hidden="true" />
 ```
 
-### Inapplicable
+### Failed
 
-#### Inapplicable example 1
+#### Failed example 1
 
-decorative image.
+No accessible name and is not marked as [decorative](#decorative)
 
 ```html
-<img alt="" />
+<img />
 ```
 
-#### Inapplicable example 2
+#### Failed example 2
 
-decorative image.
+Non-image element with image role but no accessible name
 
 ```html
-<img role="presentation" />
+<div role="img"></div>
 ```
 
-#### Inapplicable example 3
+#### Failed example 3
 
-`img` element with no role.
+Image element inside a `div` positioned off screen with no accessible name and is not marked as [decorative](#decorative)
 
 ```html
-<img role="none" />
+<div style="margin-left:-9999px;"><img /></div>
 ```
 
-#### Inapplicable example 4
+#### Failed example 4
 
-Non-image element.
+HTML `img` element is not [included in the accessibility tree](#included-in-the-accessibility-tree), does not have accessible name and is not marked as decorative
```

Comment:
This should be inapplicable because the `img` element is not included in the accessibility tree